.github/workflows/test-application.yaml

```
@@ -0,0 +1,43 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }} ${{ matrix.dependencies}}'
```

we actually never set the symfony version in this build matrix.

```suggestion
        name: 'PHP ${{ matrix.php-version }} ${{ matrix.dependencies}}'
```